### PR TITLE
Add SV rank model.

### PR DESCRIPTION
**Pull request by dnil**
Corollary to CG PR #346, Scout Issue #493, Scout PR #1220.

I didn't see any obvious error handling, so ducked in the sand without raising exceptions - is the intention to simply fail early?

- [x] code reviewed by @ingkebil 
- [x] tests ok by @emiliaol 
- [x] deploy ok by @ingkebil 

This is minor version: adds functionality.

### CODE REVIEW

**Comment by ingkebil** [comment]
Hi Daniel! Thanks a bunch!

Yes, we try to fail early .. and clearly. We will rewrite most of this when we import it into CG later this year, so whatever the standard was before, mimicking it sounds like a plan.

We'll have a look at all PRs later (sorry a bit tight trying to resolve the SWEDAC deviations)

File: trailblazer/mip/files.py [hunk 1]
```diff
@@ -52,6 +52,7 @@ def parse_sampleinfo(data: dict) -> dict:
         'family': data['family'],
         'genome_build': genome_build_str,
         'rank_model_version': data['program']['genmod']['rank_model']['version'],
+        'sv_rank_model_version': data['program']['genmod']['sv_rank_model']['version']
```
Comment (ingkebil) [inline trailblazer/mip/files.py]:
```suggestion
        'sv_rank_model_version': data['program']['genmod']['sv_rank_model']['version'],
```

Comment (dnil) [reply trailblazer/mip/files.py]: :+1:

File: trailblazer/mip/files.py [hunk 2]
```diff
@@ -52,6 +52,7 @@ def parse_sampleinfo(data: dict) -> dict:
         'family': data['family'],
         'genome_build': genome_build_str,
         'rank_model_version': data['program']['genmod']['rank_model']['version'],
+        'sv_rank_model_version': data['program']['genmod']['sv_rank_model']['version'],
```
Comment (dnil) [inline trailblazer/mip/files.py]:
```suggestion
        'sv_rank_model_version': data['program']['sv_genmod']['sv_rank_model']['version'],
```
As per @emiliaol's suggestion!

**Comment by emiliaol** [comment]
Tested in https://github.com/Clinical-Genomics/cg/pull/346 with great success!

**Pull request merged**